With clang 16 and libcxx this is causig build error such as:
/usr/include/exiv2/value.hpp:54:17: error: no template named 'auto_ptr' in namespace 'std'; did you mean simply 'auto_ptr'?
        typedef std::auto_ptr<Value> AutoPtr;
Reverting upstream commit https://gitlab.gnome.org/GNOME/gexiv2/-/commit/2eb5c2c9ac1d36bee7089117e1f4825162a9409d
helps solve them for now.

There is on going work for exiv2-0.28
--- a/meson.build
+++ b/meson.build
@@ -5,7 +5,7 @@ project(
     license: 'GPL-2.0-or-later',
     meson_version : '>=0.51',
     default_options : [
-        'cpp_std=c++17'
+        'cpp_std=c++11'
     ]
 )
 
